src/App.tsx

```diff
@@ -38,79 +37,56 @@ export class App extends React.Component {
 
```

Comment:
As a rule of thumb, you can destructure all required properties from the state here. 

`{ selectedGood } = this.state` to avoid using this.state.... each time you need some data from state at your JSX.